lib/index.coffee

```diff
@@ -4,10 +4,19 @@ GotoView = require('./goto-view')
 
 module.exports =
 
-  configDefaults:
-    logToConsole: false
-    moreIgnoredNames: ''
-    autoScroll: true
+  config:
+    logToConsole:
+      default: false
+      type: 'boolean'
+      description: 'Turn on logging debug information for goto commands'
```

Comment:
Could you update this description to `Enable debug information logging for goto commands`?